flexget/plugins/clients/transmission.py

```diff
@@ -123,13 +123,13 @@ def check_seed_limits(self, torrent, session):
 
         if torrent.seedIdleMode == 1:  # use torrent's own idle limit
             idle_limit_ok = (
-                torrent.date_active + timedelta(minutes=torrent.seedIdleLimit) < datetime.now()
+                torrent.date_active + timedelta(minutes=torrent.seedIdleLimit) < datetime.now(torrent.date_active.tzinfo)
```

Comment:
I think rather than using the timezone returned from transmission, we should use the system timezone. This way things still hold up even if the transmission daemon is running in a different timezone than flexget.
```suggestion
                torrent.date_active + timedelta(minutes=torrent.seedIdleLimit) < datetime.now().astimezone()
```